src/components/Input/index.js

```diff
@@ -52,6 +63,7 @@ const InputField = ({ defaultValue, fieldData, name, ...wrapProps }) => {
             value: regex,
             message: regex && getFieldError(fieldData, strings),
           },
+          ...getMinMaxRangeErrors(rangeMin, rangeMax, strings, errorMessage),
         })}
       />
```

Comment:
please add instruction on initial render if range is added, make sure to hide it when errror is shown